src/v/cluster/topic_properties.h

```diff
@@ -221,7 +225,8 @@ struct topic_properties
           flush_bytes,
           remote_label,
           remote_topic_namespace_override,
-          iceberg_enabled);
+          iceberg_enabled,
+          leaders_preference);
```

Comment:
this didn't need to cause a bump to serde::version because it's already been done once for this release.

do we have a good way to signal this situation to a reviewer? for example, did the review not highlight this because it wasn't noticed, or because everyone knew it was fine?